cmd/agent/main.go

```diff
@@ -1,21 +1,48 @@
 package main
 
 import (
-	"time"
+	"context"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
 
-	"github.com/labstack/gommon/log"
 	"github.com/rombintu/goyametricsv2/internal/agent"
 	"github.com/rombintu/goyametricsv2/internal/config"
+	"github.com/rombintu/goyametricsv2/internal/logger"
+	"go.uber.org/zap"
 )
 
 func main() {
-	// Еще попытка. Гитхаб лагает
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Стоит завершать контекст при завершении приложения.
```suggestion
	ctx, cancel := context.WithCancel(context.Background())
	defer cancel()
```